Material.Styles/Themes/MaterialThemeBase.cs

```diff
@@ -130,19 +168,20 @@ public MaterialThemeBase(IServiceProvider serviceProvider)
             }
         });
 
-        private Task UpdateThemeAsync(ITheme? oldTheme, ITheme newTheme) {
-            return Task.WhenAll(
+        private Task UpdateThemeAsync(ITheme? oldTheme, ITheme newTheme)
```

Comment:
I've updated theme applying in dev branch

Reply:
wait let me do some changes for this code